File: .sdk/tm/php/utility/MakeContext.php
<?php
declare(strict_types=1);

// KonkanRailwayLivePosition SDK utility: make_context

require_once __DIR__ . '/../core/Context.php';

class KonkanRailwayLivePositionMakeContext
{
    public static function call(array $ctxmap, ?KonkanRailwayLivePositionContext $basectx): KonkanRailwayLivePositionContext
    {
        return new KonkanRailwayLivePositionContext($ctxmap, $basectx);
    }
}

File: .sdk/tm/py/pkg/utility/make_context.py
# KonkanRailwayLivePosition SDK utility: make_context

from projectname_sdk.core.context import KonkanRailwayLivePositionContext


def make_context_util(ctxmap, basectx):
    return KonkanRailwayLivePositionContext(ctxmap, basectx)
